app/assets/stylesheets/_variables.scss

```
@@ -0,0 +1,2 @@
+$gap: 5px;
+$totalRows: 101;
```

```suggestion
$total-rows: 101;
```

Apparently hyphenated names are a CSS convention even if other types are allowed.